HW8/Game/src/Character.java

```diff
@@ -0,0 +1,57 @@
+public class Character {
+    String name;
+    int health;
+    int strength;
+    int defence;
+    static int CHARACTER_COUNT;
+
+
+
+    public Character(){
+        CHARACTER_COUNT++;
+    }
+    void attack (Character otherCharacter){
+        if (strength - otherCharacter.defence >= 0) {
+            otherCharacter.health = otherCharacter.health - (strength - otherCharacter.defence);
```

Comment:
This works, although is a bit non-OOP like as we're changing the property of `otherCharacter` remotely.

Better choice would be to calculate amount of damage this character will give and just call `otherCharacter.takeDamage(damage)` from the target object because "Taking damage" is a behavior of those who receives the damage, not of the one who gives it